src/main/java/com/auctionapp/api/model/dto/UserDto.java

```diff
@@ -56,4 +62,40 @@ public Timestamp getCreatedAt() {
 	public Timestamp getUpdatedAt() {
 		return updatedAt;
 	}
+
+	public UserRole getRole() {
+		return role;
+	}
+
+	public void setRole(UserRole role) {
+		this.role = role;
+	}
+
+	public void setId(UUID id) {
+		this.id = id;
+	}
```

Comment:
Getters and setters should be grouped (`getX` then `setX`, etc.). Curious as to why you need setters in a DTO?